react/components/shared/DataEntry.tsx

```diff
@@ -2,10 +2,10 @@ import React, { FunctionComponent } from 'react'
 
 const DataEntry: FunctionComponent<Props> = ({ label, children }) => {
   return (
-    <div>
+    <>
```

Comment:
I have a better proposal, why don't you add prop, cssHandle and pass the css handle to here like:

```suggestion
    <div className={cssHandle}>
```